andela/wg-justice-league
# #154240257 Update nutrition diary

#### What does this PR do?
This PR adds a provision for users to add a meal they consumed in the nutrition diary

#### Description of Task to be completed?
Currently a user can enter into the nutrition plan the food they plan to eat.

We should upgrade this for them to be able to enter the food he/she actually ate during the day,in addition to what was planed with the nutritional plan.

#### How should this be manually tested?
- Assuming you have wger running on your machine
- Navigate to Nutrition > Nutrition Plans
- You'll see two tables, one for planned meal and another for meal consumed

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
Upgrade the nutrition diary feature ID:#154240257

#### Screen Shot

<img width="797" alt="screen shot 2018-02-01 at 14 36 59" src="https://user-images.githubusercontent.com/21032089/35676617-772ac586-075d-11e8-9d8d-bc6b67a0e7eb.png">
